test/test-v8debugapi.ts

```diff
@@ -802,10 +834,12 @@ describe('v8debugapi', function() {
           const item = stringItems[0];
           assert(item.status.description.format.match(
             'Only first.*config.capture.maxStringLength=3.*of length 11.'));
-          api.clear(bp);
           config.capture.maxDataSize = oldMaxData;
```

Comment:
The same concern about moving these lines inside the callback of `api.clear` as described above.